packages/browser-runtime-core/src/interpreter/interpreter.ts

```diff
@@ -26,10 +26,12 @@ export class Interpreter {
   }
 
   async evaluate(code: string): Promise<ShellResult> {
-    let result;
+    let result = toShellResult(undefined);
```

Comment:
I'm not sure about this one either (see the comment about `LAST_EXPRESSION_CALLBACK_FUNCTION_NAME `)

Reply:
Right … looking into it a bit more, I think this was never supposed to return a `Promise` or a `ShellResult` anyway – it’s all lies… ;)